lib/player.rb

```diff
@@ -1,9 +1,8 @@
 # require_relative 'main.rb'
```

Comment:
- [ ] [REQUIRED] Please remove all comments from your codebase. 
Debugging instructions are not allowed in committed code (as discussed [here](https://questions.microverse.org/t/debugging-instructions-code/346)) ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/ruby/section/debugging_requirement.png) from Pathwright)